App/Sagas/PhotoViewingSagas.ts

```diff
@@ -138,7 +139,9 @@ export function* refreshThreads(
        */
       const isSharedThread = thread.key.indexOf('textile_photos-shared') === 0
       if (isSharedThread) {
-        yield put(PhotoViewingActions.insertThread(thread))
+        const valid = thread.headBlocks.length > 0
```

Comment:
Here is the check that determines if the alert is shown to the user. Sufficient @sanderpick ?